guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 .Procedure
 . Navigate to *Hosts* > *Register Host*.
+. Optional: Select a different *Organization*.
+. Optional: Select a different *Location*.
 . Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: Operating System, Activation Key(s) and Lifecycle environment.
-. From the *Operating System* list, select the operating system of hosts that you want to register.
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
-. Optional: *Insecure* - This makes the first call insecure, however, during this first call, hosts download the CA file from {Project}. Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+Fields that inherit value from *Host group*: Operating system, Activation Key(s) and Lifecycle environment.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+ifdef::satellite[]
+You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
+endif::[]
+. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
+ifdef::satellite[]
+Specifying an operating system is required when you register machines without subscription-manager, such as Debian or Ubuntu.
```

Comment:
```suggestion
Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
```

Are you sure you want to mention Debian and Ubuntu in Satellite docs? Did you mean to go with `ifndef`?

Reply:
Good point, thanks! ifndef is probably a way to go.